src/lessons_to_events.py

```diff
@@ -50,6 +52,18 @@ def _raw_lesson_to_location(raw_lesson: dict):
     return result if result else None
 
 
+def _raw_lesson_to_uuid(raw_lesson: dict):
+    elements = [
+        raw_lesson["date"],
+        raw_lesson["time_start"],
```

Comment:
```suggestion
        raw_lesson["time_start"],
        raw_lesson["time_end"],
        raw_lesson["type"],
```
Как считаешь, резонно ли? Два разных события, но с одинаковыми названиями и начинающихся в один момент... Не знаю, возможно ли такое, но мб лучше перестраховаться?

Reply:
Звучит излишне, поскольку система не дает построить расписание с пересечениями